genesis/process/sovereignGenesisBlockCreator.go

```diff
@@ -77,6 +103,12 @@ func (gbc *sovereignGenesisBlockCreator) createSovereignEmptyGenesisBlocks() (ma
 	return mapEmptyGenesisBlocks, nil
 }
 
+func createSovereignGenesisConfig() config.EnableEpochs {
+	cfg := createGenesisConfig()
+	cfg.ESDTMultiTransferEnableEpoch = 0
```

Comment:
I think we could return error if the config is wrong. instead of setting it to 0 here.

Reply:
This is not the loaded config from config.toml.
This is a "mocked" created config for sovereign bootStraping at genesis.